docs/docs/code/python/README.md

```diff
@@ -19,6 +19,26 @@ However, you can't connect accounts, return HTTP responses, or take advantage of
 2. Click **Custom Code**
 3. In the new step, select the `python` language runtime in language dropdown
 
+## Python Code Step Structure
+
+A new Python Code step will have the following structure, with a `hander` method and a `pd` argument passed into it:
+
+```python
+
+def handler(pd: "pipedream"):
+  # Prints "Hello, World!" to the step's logs
+  print("Hello, World!")
```

Comment:
Since the logs tab isn't highlighted in the results by default, you might consider modifying these examples to export the data using `pipedream.script_helpers`

Reply:
Good point, but it seems that in this new `handler` paradigm we've included a nice `pd.export()` method instead.